core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -71,6 +71,8 @@
   private final List<Application<?>> applications;
   private final SslContextFactory sslContextFactory;
 
+  private static int threadPoolRequestQueueCapacity;
```

Comment:
nit: If this is being returned by a public API, it probably should be `volatile`.